src/Bicep.Core/LanguageConstants.cs

```diff
@@ -257,6 +257,7 @@ public static class LanguageConstants
         public static readonly TypeSymbol StringFilePath = TypeFactory.CreateStringType(validationFlags: TypeSymbolValidationFlags.IsStringFilePath);
         public static readonly TypeSymbol StringJsonFilePath = TypeFactory.CreateStringType(validationFlags: TypeSymbolValidationFlags.IsStringFilePath | TypeSymbolValidationFlags.IsStringJsonFilePath);
         public static readonly TypeSymbol StringYamlFilePath = TypeFactory.CreateStringType(validationFlags: TypeSymbolValidationFlags.IsStringFilePath | TypeSymbolValidationFlags.IsStringYamlFilePath);
+        public static readonly TypeSymbol StringDirectoryPath = TypeFactory.CreateStringType(validationFlags: TypeSymbolValidationFlags.IsStringFilePath | TypeSymbolValidationFlags.IsStringDirectoryPath);
```

Comment:
Should this be `TypeSymbolValidationFlags.IsStringDirectoryPath` instead of `TypeSymbolValidationFlags.IsStringFilePath | TypeSymbolValidationFlags.IsStringDirectoryPath`?